qp_shotgun/humann2/humann2.py

```diff
@@ -91,7 +91,8 @@ def generate_humann2_analysis_commands(forward_seqs, reverse_seqs, map_file,
             % ', '.join(sn_by_rp.keys()))
 
     cmds = []
-    params = ['--%s "%s"' % (k, v) for k, v in viewitems(parameters) if v]
+    params = ['--%s "%s"' % (k, v) if v is not True else '--%s' % k
+              for k, v in viewitems(parameters) if v]
```

Comment:
Note that I've had problems with the integer 1 here in this context...

Here's my current workaround:

https://github.com/tanaes/qp-shotgun/blob/adding-kneaddata-pipeline/qp_shotgun/kneaddata/kneaddata.py#L103